src/tools/schema2kotlin.ts

```diff
@@ -463,6 +465,9 @@ Schema(
   }
 
   generatePredicates() {
+    if (!this.node.schema) {
+      this.refinement = `{ data -> {}}`;
```

Comment:
Why is this better then the default `{ _ -> true }` ?

Reply:
Forgot to check for the default; thanks.